_el/partner.md

```diff
@@ -1,323 +1,141 @@
 ---
 permalink: "/en/partner/"
-title: "Partnerships"
-objectives:
-- Explain why teachers in schools and universities do and don't
-  adopt better teaching practices.
-- Summarize methods that can be used to effect changes in
-  educational institutions.
+title: "Συνεργασίες"
+Μετά από τη μελέτη αυτού του κεφαλαίου θα είστε σε θέση:
+- Να εξηγείτε γιατί οι δάσκαλοι στα σχολεία και στα πανεπιστήμια υιοθετούν ή δεν υιοθετούν βελτιωμένες τεχνικές διδασκαλίας. 
```

Comment:
υιοθετούν ή δεν υιοθετούν -> υιοθετούν ή όχι

Reply:
@fpsom Thanx! Corrected in the new commit